### DISCUSSION TITLE
any chance getting this into RHEL or EPEL?

### DISCUSSION THREAD

**Discussion by OLDlayeratwork** [FiloSottile/age#586] [General]
Being Rocky Linux based, it would be nice to get `age` from some official repo. Thanks.

**Comment by alerque** [comment]
Official distro packaging is usually an issue you need to take up with the distro itself. Many distros have issue trackers or other mechanisms for requesting packages. Some also have contribution workflows.

This isn't something upstream projects like this one are generally responsible for unless they just *happen* to have a team member who is also a distro maintainer.

**Comment by OLDlayeratwork** [comment]
@alerque I agree completely, but I figured the people in this project have more pull with distros than I do.  Also, I'm not a RHEL customer, so I don't have access at all.

Perhaps the author(s) knows someone at Red Hat and can fast track something.

**Comment by FiloSottile** [comment]
There is a Fedora package and this is the process to get it pulled into EPEL. https://docs.fedoraproject.org/en-US/epel/epel-package-request/

I’d be happy for someone to get that started. If it gets stuck I can reach out to contacts to inquire.

**Comment by alexconlin** [comment]
I filed a request to get it pulled into EPEL here: https://bugzilla.redhat.com/show_bug.cgi?id=2327771
